ParisBNF/abb/BNFabb72.xml

```diff
@@ -29,22 +32,607 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                     </item> 
+                     
+                     <item xml:id="a3">
+                        <locus from="179ra" to="179rb"/>
+                        <desc type="Genealogy">A genealogy called in the beginning pen trial</desc>
+                        <q xml:lang="gez">ፈቲነ፡ ብርዕ፡ በዕለተ፡ ገድና፡ ማርያም፡ ዳንኤል፡ ተስፋይ፡ ተክለ፡ ዮሐንስ፡ <add place="above">ተስፎም፡</add>
+                           አስገደ፡ ጓል፡ ቃይሕ፡ ሹም፡ ደራ፡ ካልባይ፡ ስንአፈ፡ አዕዋተ፡ ሮምዘመዳ፡ ተወልደት፡ ቃውሎስ፡<gap reason="ellipsis" resp="DN"/></q>
+                     </item> 
+                     
+                     <item xml:id="a4">
+                        <locus from="179ra" to="179rb"/>
+                        <desc type="Genealogy">A short genealogy of Christ, the line of St Mary</desc>                        
+                     </item> 
+                        
+                     <item xml:id="a5">
+                        <locus target="#179ra"/>
+                        <desc type="Unclear">A directive prescribing what to do with <foreign xml:lang="gez">tabot</foreign>
+                          defiled by <foreign xml:lang="amh">qalāṭi</foreign> ('a reckless one'?)</desc>
+                        <q xml:lang="gez">ሶበ፡ ለከፎ፡ ቀላጢ ለታቦት፡ ዘኢመፍትው፡ በምንተኒ፡ ከመዝ፡ ግበር፡ እንበለ፡ ያእምሩ፡ ሕዝብ። መቅደመ፡ ኵሉ፡
+                           ያዕርግ፡ <del rend="expunctuated">ያ</del>ዕጣነ፡ ወይድግም፡ ጸሎተ፡ ዕጣን፡ ጸዊሮ፡ ታቦተ፤<gap reason="ellipsis" resp="DN"/></q>
+                     </item> 
+                     
+                     <item xml:id="a6">
+                        <locus from="179ra" to="179rb"/>
+                        <desc type="Unclear">A directive prescribing which texts should be read daily in the church</desc>
+                      </item> 
+                     
+                     <item xml:id="a8">
+                        <locus target="#185rb"/>
+                        <desc type="OwnershipNote">An ownership note mentions the donor <persName ref="PRS14384WalattaPetros" role="donor">Walatta Ṗeṭros</persName></desc>
+                        <q xml:lang="gez">ዝመጽሐፍ፡ ዘ<placeName ref="INS1011AratoMaryamMagdalawit">ቅድስት፡ መግደላዊት፡ ዘዓራቶ፡</placeName> ዘወሀበታ፡ <persName ref="PRS14384WalattaPetros" role="donor">ወለተ፡ ጴጥሮስ፡</persName> እኅቱ፡ 
+                           ለ<persName  ref="PRS14383WaldaGiyorgis">አቤቶ፡ ወልደ፡ ጊዮርጊስ፡</persName>
+                           ወአቡሆሙ፡ ልብነ፡ ድንግል፡ ከመ፡ ይኩኖሙ፡ መድኃኒተ፡ ነፍሶሙ፨</q>
+                     </item> 
+                     
+                     <item xml:id="a9">
+                        <locus from="185rb" to="189rb"/>
+                        <desc type="Unclear">An unidentified text, possibly a directive, composed in part from quotations</desc>
+                        <q xml:lang="gez">ተዓብዮ፡ ነፍስየ፡ ለእግዚአብሔር፡ በል፡ ፫ጊዜ፡ ንዒ፡ ዘትነብሪ፡ በአርያም፡ ወንዒ፡፨<gap reason="ellipsis" resp="DN"/></q>
+                        <note>Numerous lacunas are meant possibly for the rubricated name of <persName ref="PRS6819Mary">St Mary</persName></note>
```

Comment:
It's better to include the content of the note inside desc

Reply:
ok